File: C#.Net/Tests/JDI Tests/Epam UIObjects/DataProviders/AttendeesProvider.cs
using System.Collections.Generic;
using JDI_Tests.Entities;

namespace JDI_Tests.Epam_UIObjects.DataProviders
{
    public class AttendeesProvider
    {
        public static IEnumerable<Attendee> Attendees
        {
            get
            {
                yield return new Attendee
                {
                    City = "Saint-Petersburg",
                    Comment = "I WANT TO WORK IN EPAM!!!",
                    Country = "Russian Federation",
                    Cv = "my_cv.pdf",
                    Email = "roman_iovlev@epam.com",
                    Filter = new JobSearchFilter(),
                    LastName = "Iovlev",
                    Name = "Roman"
                };
                yield return new Attendee
                {
                    City = "Mumbai",
                    Comment = "I also want to work in Epam",
                    Country = "India",
                    Cv = "new_document.txt",
                    Email = "Kamal_Haasan@example.com",
                    Filter = new JobSearchFilter(),
                    LastName = "Haasan",
                    Name = "Kamal"
                };
            }
        }
    }
}

File: Java/Tests/jdi-uitest-tutorialtests/src/main/java/com/epam/jdi/dataProviders/AttendeesProvider.java
package com.epam.jdi.dataProviders;

import com.epam.jdi.entities.Attendee;
import org.testng.annotations.DataProvider;


/**
 * Created by Roman_Iovlev on 9/2/2015.
 */
public final class AttendeesProvider {
    private AttendeesProvider() { }

    @DataProvider(name = "attendees")
    public static Object[][] attendees() {
        return new Object[][]{
                { new Attendee() }
        };
    }
}
